servers/physics/collision_object_sw.h

```diff
@@ -86,13 +86,9 @@ class CollisionObjectSW : public ShapeOwnerSW {
 	void _unregister_shapes();
 
 	_FORCE_INLINE_ void _set_transform(const Transform &p_transform, bool p_update_shapes = true) {
-
 #ifdef DEBUG_ENABLED
 
-		if (p_transform.origin.length_squared() > MAX_OBJECT_DISTANCE_X2) {
-			ERR_EXPLAIN("Object went too far away (more than " + itos(MAX_OBJECT_DISTANCE) + "mts from origin).");
-			ERR_FAIL();
-		}
+		ERR_FAIL_COND_MSG(p_transform.origin.length_squared() > MAX_OBJECT_DISTANCE_X2, "Object went too far away (more than " + itos(MAX_OBJECT_DISTANCE) + "mts from origin).");
```

Comment:
I'd add a space before "mts". And I'd change "mts" for "m" because that's not a standard unit symbol. Or use "units" instead?